app/models/form.rb

```diff
@@ -37,6 +37,6 @@ def draft_belonging_to?(user)
   end
 
   def form_name
-    name.parameterize
+    name.try(:parameterize)
```

Comment:
Confused by this as name is validated for presence.

Reply:
Correct, and I think this may not be a problem in production given the html validations we have in place. However, this method is called on `new` objects as well as persisted objects--because of the way friendly-id works, it tries to create a slug immediately. This will probably never be an issue in production, but when I was playing in the console while developing I noticed this:
```
f = Form.new
=> #<Form id: nil, name: nil, intro_text: nil, created_at: nil, updated_at: nil, email: nil, reply_to: nil, slug: nil>
f.valid?
Traceback (most recent call last):
        2: from (irb):4
        1: from app/models/form.rb:40:in `form_name'
NoMethodError (undefined method `parameterize' for nil:NilClass)
```
Uncool, in my opinion, because it prevents the model validations from working as intended. Although you can certainly run all the current tests without that nil-handling, I believe it makes developing nicer and might prevent some weirdness later.

Reply:
OK, makes sense and agreed. My concern here was just that IIRC we use the parameterized name for the routing? So where I was going with this is that it could do weird things to the URL if having no name while you're making a draft for a new form is part of the new workflow.
As long as this is just to accommodate dev/test, then no worries.

Reply:
Yeah, it would definitely do weird things. However the form, like you said, can't be saved without a name regardless. 
This will not only accommodate dev/test, but allow model validations to work properly in the absence of html validations (an unlikely scenario) in production.

Reply:
OK, cool